motr/client_init.c

```diff
@@ -1678,7 +1683,8 @@ int m0_client_init(struct m0_client **m0c_p,
 	}
 
 	if (conf->mc_is_addb_init) {
-		char buf[64];
+		char buf[128];
```

Comment:
Is 128 bytes enough to hold the location, mostly we will have location like /etc/cortx/log/<Machine id>/add_stob_dir

Reply:
![image](https://user-images.githubusercontent.com/66949386/185840843-bb7ad0fa-61ea-41ef-acc8-35689fc2f87c.png)

Reply:
@rahul27kumar, I thought in our conversation we agreed to change the buffer size to take MAXPATHLEN.